api-2.7/src/main/java/org/openmrs/module/initializer/api/conceptreferencerange/ConceptReferenceRangeLineProcessor.java

```
@@ -0,0 +1,50 @@
+package org.openmrs.module.initializer.api.conceptreferencerange;
+
+import org.openmrs.api.ConceptService;
+import org.openmrs.module.initializer.api.BaseLineProcessor;
+import org.openmrs.module.initializer.api.CsvLine;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Component;
+import org.openmrs.ConceptReferenceRange;
+
+@Component
+public class ConceptReferenceRangeLineProcessor extends BaseLineProcessor<ConceptReferenceRange> {
+	
+	private final String HEADER_CONCEPT_NUMERIC_UUID = "Concept Numeric UUID";
+	
+	private final String HEADER_ABSOLUTE_LOW = "Absolute Low";
+	
+	private final String HEADER_ABSOLUTE_HIGH = "Absolute High";
+	
+	private final String HEADER_CRITICAL_LOW = "Critical Low";
+	
+	private final String HEADER_CRITICAL_HIGH = "Critical High";
+	
+	private final String HEADER_NORMAL_LOW = "Normal Low";
+	
+	private final String HEADER_NORMAL_HIGH = "Normal High";
+	
+	private final String HEADER_CRITERIA = "Criteria";
+	
+	private final ConceptService conceptService;
+	
+	public ConceptReferenceRangeLineProcessor(@Qualifier("conceptService") ConceptService conceptService) {
+		this.conceptService = conceptService;
+	}
+	
+	public ConceptReferenceRange fill(ConceptReferenceRange conceptReferenceRange, CsvLine line)
+	        throws IllegalArgumentException {
+		conceptReferenceRange
```

This should have `line.get(HEADER..., true)` since it's an error is the UUID isn't set, right? It also should raise an error if the concept couldn't be found by UUID or if the concept found by that UUID isn't a concept numeric. Can we add some tests for this?